### PR TITLE
New Package: WaterbugStudios.Mazerator v0.0.1

### PR DESCRIPTION
**Pull request by Trenly**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
  - Closes #40971 
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/40997)

### CODE REVIEW

**Comment by Trenly** [comment]
@OfficialEsco  I can't quite figure out how to get past this. Any ideas? 
image

File: manifests/w/WaterbugStudios/Mazerator/0.0.1/WaterbugStudios.Mazerator.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: WaterbugStudios.Mazerator
+PackageVersion: 0.0.1
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
```
Comment (OfficialEsco) [inline manifests/w/WaterbugStudios/Mazerator/0.0.1/WaterbugStudios.Mazerator.installer.yaml:8]:
```suggestion
  InstallerType: inno
```

File: manifests/w/WaterbugStudios/Mazerator/0.0.1/WaterbugStudios.Mazerator.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  InstallerUrl: https://download.tuxfamily.org/mazerator/Mazerator/v0.0.1/mazerator-0.0.1-x64.exe
+  InstallerSha256: 7653007BA64DE7EC9393D8944018CCAB2162F5C411671EBA19E86D92282B91A9
+  InstallerSwitches:
+    Custom: /ALLUSERS
+  Scope: machine
+  ProductCode: '{A2086546-A47B-486A-9449-DC433173B399}_is1'
+- Architecture: x64
+  InstallerType: nullsoft
```
Comment (OfficialEsco) [inline manifests/w/WaterbugStudios/Mazerator/0.0.1/WaterbugStudios.Mazerator.installer.yaml:16]:
```suggestion
  InstallerType: inno
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
*cough* yes.

**Comment by Trenly** [comment]
> _cough_ yes.

🤦

**Comment by Trenly** [comment]
> https://github.com/microsoft/winget-pkgs/pull/40997/commits/d792f0cf6ddcab00b4cf93e4df55401c8d3e692e

Oof, its been a long day

**Review by OfficialEsco — APPROVED** [review]
> Edit: Didn't work

> Oof, its been a long day


I can see that... 🤔😁
```
Enter a pull request number: #40997

Manifest validation succeeded.
Found Mazerator [WaterbugStudios.Mazerator] Version 0.0.1
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://download.tuxfamily.org/mazerator/Mazerator/v0.0.1/mazerator-0.0.1-x64.exe
  ██████████████████████████████  48.9 MB / 48.9 MB
Successfully verified installer hash
Starting package install...
Successfully installed

Name        : Mazerator version 0.0.1
Publisher   : Waterbug Studios
Version     : 0.0.1
ProductCode : {A2086546-A47B-486A-9449-DC433173B399}_is1
Uninstall   : winget uninstall "{A2086546-A47B-486A-9449-DC433173B399}_is1"

Press any key to continue . . .
```

**Pull request merged**